crates/modelardb_server/src/context.rs

```diff
@@ -146,23 +146,33 @@ impl Context {
             .create_model_table(&model_table_metadata.name)
             .await?;
 
-        let table_metadata_manager = self
+        let query_table_metadata_manager = self
```

Comment:
Maybe `query_folder_table_metadata_manager` as I did not understand what `quey_` was until I read how the variable was created.

Reply:
Fixed.